chemprop/data/collate.py

```diff
@@ -85,7 +85,7 @@ def collate_batch(batch: Iterable[Datum]) -> TrainingBatch:
 
     return TrainingBatch(
         BatchMolGraph(mgs),
-        None if V_ds[0] is None else torch.from_numpy(np.stack(V_ds, axis=0)).float(),
+        None if V_ds[0] is None else torch.from_numpy(np.concatenate(V_ds)).float(),
```

Comment:
@davidegraff Hi David, I made this change here but I would like to double check. V_d is of shape V x d_vd, where V is the number of atoms in the molecule and d_vd is the dimension of the additional atom descriptors, so we are not able to stack the V_ds. Instead, we can concatenate on the 0th axis. Is this understanding correct?